src/frontend/magnify/src/services/http/http.service.ts

```diff
@@ -81,3 +89,7 @@ MagnifyApi.interceptors.response.use(
     return Promise.reject(error);
   },
 );
+
+setTimeout(() => {
+  HttpService.setRequestLanguage(getLocaleStorageLang());
+}, 0);
```

Comment:
Why do you need to do this ? We could set a default headers like you did to set the `Authorization` header.
Why not use an IIFE instead of this "instant" timeout.